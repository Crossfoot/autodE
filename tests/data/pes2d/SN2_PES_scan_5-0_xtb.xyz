6

F   -2.52958  -0.14748  0.04659  
Cl  1.24516   0.07463   -0.01889 
C   -0.53269  -0.03102  0.00976  
H   -0.75602  -0.61733  -0.86548 
H   -0.73171  -0.51920  0.94874  
H   -0.83811  1.00032   -0.04202 
