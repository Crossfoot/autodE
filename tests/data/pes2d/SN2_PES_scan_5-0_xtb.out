      -----------------------------------------------------------      
     |                   =====================                   |     
     |                           x T B                           |     
     |                   =====================                   |     
     |                         S. Grimme                         |     
     |          Mulliken Center for Theoretical Chemistry        |     
     |                    University of Bonn                     |     
      -----------------------------------------------------------      

   * xtb version 6.2.3 (830e466) compiled by 'ehlert@majestix' on 2020-03-16

   xtb is free software: you can redistribute it and/or modify it under
   the terms of the GNU Lesser General Public License as published by
   the Free Software Foundation, either version 3 of the License, or
   (at your option) any later version.
   
   xtb is distributed in the hope that it will be useful,
   but WITHOUT ANY WARRANTY; without even the implied warranty of
   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
   GNU Lesser General Public License for more details.
   
   Cite this work as:
   * S. Grimme, C. Bannwarth, P. Shushkov, J. Chem. Theory Comput., 2017,
     13, 1989-2009. DOI: 10.1021/acs.jctc.7b00118
   * C. Bannwarth, S. Ehlert and S. Grimme., J. Chem. Theory Comput., 2019,
     15, 1652-1671. DOI: 10.1021/acs.jctc.8b01176
   * P. Pracht, E. Caldeweyher, S. Ehlert, S. Grimme, ChemRxiv, 2019, preprint.
     DOI: 10.26434/chemrxiv.8326202.v1
   
   for DFT-D4:
   * E. Caldeweyher, C. Bannwarth and S. Grimme, J. Chem. Phys., 2017,
     147, 034112. DOI: 10.1063/1.4993215
   * E. Caldeweyher, S. Ehlert, A. Hansen, H. Neugebauer, S. Spicher,
     C. Bannwarth and S. Grimme, J. Chem. Phys., 2019, 150, 154122.
     DOI: 10.1063/1.5090222
   
   for sTDA-xTB:
   * S. Grimme and C. Bannwarth, J. Chem. Phys., 2016, 145, 054103.
     DOI: 10.1063/1.4959605
   
   in the mass-spec context:
   * V. Asgeirsson, C. Bauer and S. Grimme, Chem. Sci., 2017, 8, 4879.
     DOI: 10.1039/c7sc00601b
   
   for metadynamics refer to:
   * S. Grimme, J. Chem. Theory Comput., 2019, 155, 2847-2862
     DOI: 10.1021/acs.jctc.9b00143
   
   with help from (in alphabetical order)
   C. Bannwarth, F. Bohle, G. Brandenburg, E. Caldeweyher, M. Checinski,
   S. Dohm, S. Ehlert, S. Ehrlich, F. März, H. Neugebauer, J. Pisarek,
   P. Pracht, P. Shushkov, and S. Spicher.
   
 * started run on 2020/04/08 at 23:04:04.701     

           -------------------------------------------------
          |                Calculation Setup                |
           -------------------------------------------------

          program call               : /home/tom/.local/bin/xtb SN2_PES_scan_5-0_xtb.xyz --chrg -1 --opt --input xcontrol_SN2_PES_scan_5-0
          coordinate file            : SN2_PES_scan_5-0_xtb.xyz
          omp threads                :                     1
          number of atoms            :                     6
          number of electrons        :                    22
          charge                     :                    -1
          spin                       :                   0.0
          first test random number   :      0.93022582264114

constraining bond 1 3 to    1.5000000 Å, actual value:    2.0006222 Å
constraining bond 2 3 to    1.7800000 Å, actual value:    1.7812168 Å

molecular fragmentation (1/2 indicates fragments):
122222
# atoms in fragment 1/2:     1     5
 fragment masses (1/2) :       19.00       50.49
CMA distance (Bohr)    :   6.117
constraining FC (au)   : 10.0000
########################################################################
[WARNING] Please study the warnings concerning your input carefully
-2- prog_main: XTBPATH is not set, using XTBHOME instead
-1- prog_main: XTBHOME is not set, using HOME instead
########################################################################
           ------------------------------------------------- 
          |                 G F N 2 - x T B                 |
          | Geometry, Frequencies, Noncovalent interactions |
          |            JCTC 2019 parametrisation            |
           ------------------------------------------------- 
             k(s)              :                1.8500
             k(p)              :                2.2300
             k(d)              :                2.2300
             k(f)              :                2.0000
             kEN (H0ij)        :               -2.0000
             D4 a1             :                0.5200
             D4 a2             :                5.0000
             D4 s6             :                1.0000
             D4 s8             :                2.7000
             D4 s9             :                5.0000
             alphaj            :                2.0000

  Z AO/shell   Hii/eV     exponent
  1     Wed Apr 25 08:07:45 CEST 2018   EN: 2.200 GM2: 0.406  GM3: 0.0800  RAES: 1.40
     1s    -10.707211    1.230000
  6     Wed Apr 25 08:07:45 CEST 2018   EN: 2.550 GM2: 0.538  GM3: 0.1500  RAES: 3.00
     2s    -13.970922    2.096432
     2p    -10.063292    1.800000
  9     Mon Apr 23 13:58:50 CEST 2018   EN: 3.980 GM2: 0.532  GM3: 0.1426  RAES: 2.40
     2s    -23.458179    2.416361
     2p    -15.746583    2.308399
 17     Mon Apr 23 14:09:22 CEST 2018   EN: 3.160 GM2: 0.249  GM3: 0.1495  RAES: 2.50
     3s    -29.278781    2.485265
     3p    -12.673758    2.199650
     3d     -0.240338    2.476089

           ------------------------------------------------- 
          |        Self-Consistent Charge Iterations        |
           ------------------------------------------------- 

          ...................................................
          :                      SETUP                      :
          :.................................................:
          :  # basis functions                  21          :
          :  # atomic orbitals                  20          :
          :  # shells                           10          :
          :  # electrons                        22          :
          :  max. iterations                   250          :
          :  Hamiltonian                  GFN2-xTB          :
          :  restarted?                      false          :
          :  GBSA solvation                  false          :
          :  PC potential                    false          :
          :  electronic temp.          300.0000000     K    :
          :  accuracy                    1.0000000          :
          :  -> integral cutoff          0.2500000E+02      :
          :  -> integral neglect         0.1000000E-07      :
          :  -> SCF convergence          0.1000000E-05 Eh   :
          :  -> wf. convergence          0.1000000E-03 e    :
          :  Broyden damping             0.4000000          :
          ...................................................

 iter      E             dE          RMSdq      gap      omega  full diag
   1    -13.1245730 -0.131246E+02  0.650E+00    6.67       0.0  T
   2    -13.1943266 -0.697537E-01  0.334E+00    4.70       1.0  T
   3    -13.1963585 -0.203185E-02  0.171E+00    4.47       1.0  T
   4    -13.1974950 -0.113650E-02  0.610E-01    4.21       1.0  T
   5    -13.1981687 -0.673736E-03  0.324E-01    4.45       1.0  T
   6    -13.1984269 -0.258126E-03  0.249E-01    4.19       1.0  T
   7    -13.1986304 -0.203570E-03  0.567E-02    4.23       1.0  T
   8    -13.1986324 -0.194145E-05  0.159E-02    4.20       2.6  T
   9    -13.1986334 -0.108231E-05  0.530E-03    4.21       7.7  T
  10    -13.1986336 -0.109316E-06  0.534E-04    4.22      76.4  T
  11    -13.1986336 -0.763245E-09  0.228E-04    4.22     178.7  T

   *** convergence criteria satisfied after 11 iterations ***

         #    Occupation            Energy/Eh            Energy/eV
      -------------------------------------------------------------
         1        2.0000           -0.8911459             -24.2493
       ...           ...                  ...                  ...
         5        2.0000           -0.3294216              -8.9640
         6        2.0000           -0.3057404              -8.3196
         7        2.0000           -0.2487870              -6.7698
         8        2.0000           -0.2487784              -6.7696
         9        2.0000           -0.1247456              -3.3945
        10        2.0000           -0.1247388              -3.3943
        11        2.0000           -0.1243559              -3.3839 (HOMO)
        12                          0.0306068               0.8329 (LUMO)
        13                          0.2008640               5.4658
        14                          0.2008663               5.4659
        15                          0.2048454               5.5741
        16                          0.2048604               5.5745
       ...                                ...                  ...
        20                          0.4275356              11.6338
      -------------------------------------------------------------
                  HL-Gap            0.1549627 Eh            4.2167 eV
             Fermi-level           -0.0468746 Eh           -1.2755 eV

 SCC (total)                   0 d,  0 h,  0 min,  0.003 sec
 SCC setup                      ...        0 min,  0.000 sec (  2.424%)
 Dispersion                     ...        0 min,  0.000 sec (  0.468%)
 integral evaluation            ...        0 min,  0.001 sec ( 25.992%)
 zeroth order Hamiltonian       ...        0 min,  0.000 sec (  0.615%)
 iterations                     ...        0 min,  0.001 sec ( 40.544%)
 molecular gradient             ...        0 min,  0.001 sec ( 26.882%)
 printout                       ...        0 min,  0.000 sec (  2.314%)

         :::::::::::::::::::::::::::::::::::::::::::::::::::::
         ::                     SUMMARY                     ::
         :::::::::::::::::::::::::::::::::::::::::::::::::::::
         :: total energy              -4.188988278630 Eh    ::
         :: gradient norm             26.725122451717 Eh/a0 ::
         :: HOMO-LUMO gap              4.216749724023 eV    ::
         ::.................................................::
         :: SCC energy               -13.198633554162 Eh    ::
         :: -> isotropic ES            0.219626563097 Eh    ::
         :: -> anisotropic ES         -0.007967004521 Eh    ::
         :: -> anisotropic XC         -0.010562027684 Eh    ::
         :: -> dispersion             -0.002201510931 Eh    ::
         :: repulsion energy           0.059700222266 Eh    ::
         :: add. restraining           8.949945067602 Eh    ::
         :::::::::::::::::::::::::::::::::::::::::::::::::::::

      ----------------------------------------------------------- 
     |                   =====================                   |
     |                        A N C O P T                        |
     |                   =====================                   |
     |               Approximate Normal Coordinate               |
     |                Rational Function Optimizer                |
      ----------------------------------------------------------- 

          ...................................................
          :                      SETUP                      :
          :.................................................:
          :   optimization level            normal          :
          :   max. optcycles                   200          :
          :   ANC micro-cycles                  20          :
          :   degrees of freedom                12          :
          :.................................................:
          :   RF solver                   davidson          :
          :   write xtbopt.log                true          :
          :   linear?                        false          :
          :   energy convergence         0.5000000E-05 Eh   :
          :   grad. convergence          0.1000000E-02 Eh/α :
          :   maximium RF displ.         1.0000000          :
          :   Hlow (freq-cutoff)         0.1000000E-01      :
          :   Hmax (freq-cutoff)         5.0000000          :
          :   S6 in model hess.         20.0000000          :
          ...................................................

generating ANC from model Hessian ...
Using Lindh-Hessian (1995)
 Shifting diagonal of input Hessian by   0.000000000000000E+000
 Lowest  eigenvalues of input Hessian
   -0.000000   -0.000000    0.000000    0.000000    0.000000    0.000000
    0.082935    0.083578    0.128751    0.128759    0.128821    0.460170
    0.652860    0.653007    8.336875    8.348012   20.256234   60.926681
 Highest eigenvalues
    0.652860    0.653007    8.336875    8.348012   20.256234   60.926681
 

........................................................................
.............................. CYCLE    1 ..............................
........................................................................
   1    -13.1986336 -0.131986E+02  0.163E-05    4.22       0.0  T
   2    -13.1986336 -0.536460E-12  0.106E-05    4.22    3836.5  T
   3    -13.1986336 -0.103029E-12  0.860E-06    4.22    4746.1  T
     SCC iter.                  ...        0 min,  0.000 sec
     gradient                   ...        0 min,  0.001 sec
 * total energy  :    -4.1889883 Eh     change       -0.1453777E-07 Eh
   gradient norm :    26.7251295 Eh/α   predicted     0.0000000E+00 (-100.00%)
   displ. norm   :     0.9108219 α      lambda       -0.2434183E+02
   maximum displ.:     0.7853522 α      in ANC's #9, #12, #6, ...

........................................................................
.............................. CYCLE    2 ..............................
........................................................................
   1    -13.1608988 -0.131609E+02  0.418E+00    7.90       0.0  T
   2    -13.1896103 -0.287115E-01  0.193E+00   10.54       1.0  T
   3    -13.1974371 -0.782683E-02  0.830E-01    9.96       1.0  T
   4    -13.1980044 -0.567238E-03  0.355E-01    9.88       1.0  T
   5    -13.1981932 -0.188883E-03  0.222E-01    9.90       1.0  T
   6    -13.1982820 -0.887926E-04  0.966E-02    9.97       1.0  T
   7    -13.1983005 -0.184353E-04  0.343E-02   10.00       1.2  T
   8    -13.1983010 -0.496191E-06  0.899E-04    9.99      45.4  T
   9    -13.1983010 -0.640160E-09  0.460E-04    9.99      88.7  T
     SCC iter.                  ...        0 min,  0.001 sec
     gradient                   ...        0 min,  0.001 sec
 * total energy  :    -7.7709105 Eh     change       -0.3581922E+01 Eh
   gradient norm :    24.5767201 Eh/α   predicted    -0.2226786E+02 ( 521.67%)
   displ. norm   :     0.5983478 α      lambda       -0.1183504E+02
   maximum displ.:     0.4280555 α      in ANC's #12, #9, #5, ...

........................................................................
.............................. CYCLE    3 ..............................
........................................................................
   1    -13.0847955 -0.130848E+02  0.300E+00    9.21       0.0  T
   2    -13.0968831 -0.120876E-01  0.175E+00    8.47       1.0  T
   3    -13.1011241 -0.424105E-02  0.105E+00    7.96       1.0  T
   4    -13.1022922 -0.116812E-02  0.731E-01    7.94       1.0  T
   5    -13.1034840 -0.119179E-02  0.329E-01    7.96       1.0  T
   6    -13.1037115 -0.227486E-03  0.373E-02    7.93       1.1  T
   7    -13.1037136 -0.209914E-05  0.179E-02    7.94       2.3  T
   8    -13.1037138 -0.248338E-06  0.212E-03    7.93      19.3  T
   9    -13.1037139 -0.863437E-08  0.662E-04    7.94      61.6  T
     SCC iter.                  ...        0 min,  0.001 sec
     gradient                   ...        0 min,  0.001 sec
 * total energy  :   -12.5536162 Eh     change       -0.4782706E+01 Eh
   gradient norm :     4.6622532 Eh/α   predicted    -0.8036112E+01 (  68.02%)
   displ. norm   :     0.4077367 α      lambda       -0.1811553E+01
   maximum displ.:     0.4020606 α      in ANC's #12, #9, #6, ...

........................................................................
.............................. CYCLE    4 ..............................
........................................................................
   1    -13.1678020 -0.131678E+02  0.149E+00    7.24       0.0  T
   2    -13.1708786 -0.307663E-02  0.894E-01    7.25       1.0  T
   3    -13.1713557 -0.477085E-03  0.526E-01    7.14       1.0  T
   4    -13.1716590 -0.303286E-03  0.295E-01    7.13       1.0  T
   5    -13.1718106 -0.151623E-03  0.209E-01    7.17       1.0  T
   6    -13.1719236 -0.112963E-03  0.284E-02    7.20       1.4  T
   7    -13.1719249 -0.130841E-05  0.943E-03    7.20       4.3  T
   8    -13.1719251 -0.170408E-06  0.198E-03    7.20      20.6  T
   9    -13.1719251 -0.128486E-08  0.602E-04    7.20      67.8  T
     SCC iter.                  ...        0 min,  0.001 sec
     gradient                   ...        0 min,  0.001 sec
 * total energy  :   -12.6076786 Eh     change       -0.5406245E-01 Eh
   gradient norm :     5.6163552 Eh/α   predicted    -0.1056361E+01 (1853.97%)
   displ. norm   :     0.1877707 α      lambda       -0.9257467E+00
   maximum displ.:     0.1662873 α      in ANC's #12, #9, #5, ...

........................................................................
.............................. CYCLE    5 ..............................
........................................................................
   1    -13.1465014 -0.131465E+02  0.812E-01    7.74       0.0  T
   2    -13.1477539 -0.125248E-02  0.385E-01    7.85       1.0  T
   3    -13.1479436 -0.189784E-03  0.172E-01    7.94       1.0  T
   4    -13.1479590 -0.153869E-04  0.925E-02    7.94       1.0  T
   5    -13.1479702 -0.111298E-04  0.564E-02    7.94       1.0  T
   6    -13.1479787 -0.856459E-05  0.782E-03    7.93       5.2  T
   7    -13.1479788 -0.113333E-06  0.397E-03    7.93      10.3  T
   8    -13.1479789 -0.115858E-07  0.576E-04    7.93      70.8  T
     SCC iter.                  ...        0 min,  0.001 sec
     gradient                   ...        0 min,  0.001 sec
 * total energy  :   -13.0790643 Eh     change       -0.4713857E+00 Eh
   gradient norm :     0.3430750 Eh/α   predicted    -0.4791934E+00 (   1.66%)
   displ. norm   :     0.2013729 α      lambda       -0.9995634E-02
   maximum displ.:     0.1496947 α      in ANC's #5, #3, #4, ...

........................................................................
.............................. CYCLE    6 ..............................
........................................................................
   1    -13.1332793 -0.131333E+02  0.382E-01    7.89       0.0  T
   2    -13.1333610 -0.817019E-04  0.205E-01    7.91       1.0  T
   3    -13.1333772 -0.162764E-04  0.698E-02    7.93       1.0  T
   4    -13.1333817 -0.449182E-05  0.456E-02    7.93       1.0  T
   5    -13.1333854 -0.367945E-05  0.298E-02    7.92       1.4  T
   6    -13.1333877 -0.225485E-05  0.509E-03    7.92       8.0  T
   7    -13.1333877 -0.255258E-07  0.743E-04    7.92      54.9  T
     SCC iter.                  ...        0 min,  0.001 sec
     gradient                   ...        0 min,  0.001 sec
 * total energy  :   -13.0815682 Eh     change       -0.2503892E-02 Eh
   gradient norm :     0.0532596 Eh/α   predicted    -0.5200496E-02 ( 107.70%)
   displ. norm   :     0.0993249 α      lambda       -0.2325011E-02
   maximum displ.:     0.0755772 α      in ANC's #5, #3, #4, ...

........................................................................
.............................. CYCLE    7 ..............................
........................................................................
   1    -13.1379063 -0.131379E+02  0.188E-01    7.89       0.0  T
   2    -13.1379282 -0.218256E-04  0.961E-02    7.88       1.0  T
   3    -13.1379318 -0.367995E-05  0.218E-02    7.86       1.9  T
   4    -13.1379323 -0.424284E-06  0.116E-02    7.87       3.5  T
   5    -13.1379325 -0.179174E-06  0.868E-03    7.87       4.7  T
   6    -13.1379327 -0.210833E-06  0.790E-04    7.87      51.7  T
   7    -13.1379327 -0.399375E-09  0.329E-04    7.87     124.2  T
     SCC iter.                  ...        0 min,  0.001 sec
     gradient                   ...        0 min,  0.001 sec
 * total energy  :   -13.0826637 Eh     change       -0.1095473E-02 Eh
   gradient norm :     0.0400141 Eh/α   predicted    -0.1173985E-02 (   7.17%)
   displ. norm   :     0.0065622 α      lambda       -0.6479389E-04
   maximum displ.:     0.0040457 α      in ANC's #5, #6, #3, ...

........................................................................
.............................. CYCLE    8 ..............................
........................................................................
   1    -13.1375921 -0.131376E+02  0.116E-02    7.86       0.0  T
   2    -13.1375923 -0.177212E-06  0.516E-03    7.86       7.9  T
   3    -13.1375923 -0.303518E-07  0.182E-03    7.87      22.4  T
     SCC iter.                  ...        0 min,  0.001 sec
     gradient                   ...        0 min,  0.001 sec
 * total energy  :   -13.0826925 Eh     change       -0.2882948E-04 Eh
   gradient norm :     0.0084109 Eh/α   predicted    -0.3239958E-04 (  12.38%)
   displ. norm   :     0.0020980 α      lambda       -0.3503423E-05
   maximum displ.:     0.0020644 α      in ANC's #6, #12, #3, ...

........................................................................
.............................. CYCLE    9 ..............................
........................................................................
   1    -13.1373331 -0.131373E+02  0.388E-03    7.86       0.0  T
   2    -13.1373331 -0.905789E-08  0.187E-03    7.86      21.9  T
   3    -13.1373331 -0.113859E-08  0.528E-04    7.86      77.3  T
     SCC iter.                  ...        0 min,  0.001 sec
     gradient                   ...        0 min,  0.001 sec
 * total energy  :   -13.0826946 Eh     change       -0.2074494E-05 Eh
   gradient norm :     0.0008402 Eh/α   predicted    -0.1750591E-05 ( -15.61%)
   displ. norm   :     0.0016265 α      lambda       -0.3828844E-06
   maximum displ.:     0.0015439 α      in ANC's #6, #5, #2, ...

   *** GEOMETRY OPTIMIZATION CONVERGED AFTER 9 ITERATIONS ***

------------------------------------------------------------------------
 total energy gain   :        -8.8937063 Eh    -5580.8850 kcal/mol
 total RMSD          :         0.3231035 a0        0.1710 Å
 total power (kW/mol):     -2594.4914086 (step)********** (real)
------------------------------------------------------------------------

 ANCopt (total)                0 d,  0 h,  0 min,  0.020 sec
 optimizer setup                ...        0 min,  0.000 sec (  0.260%)
 model hessian                  ...        0 min,  0.000 sec (  0.694%)
 ANC generation                 ...        0 min,  0.000 sec (  0.408%)
 coordinate transformation      ...        0 min,  0.000 sec (  0.207%)
 single point calculation       ...        0 min,  0.019 sec ( 93.846%)
 optimization log               ...        0 min,  0.000 sec (  1.312%)
 hessian update                 ...        0 min,  0.000 sec (  0.206%)
 rational function              ...        0 min,  0.000 sec (  1.513%)

================
 final structure:
================
6
 xtb: 6.2.3 (830e466)
F        -2.22542565278295   -0.13001053402703    0.04041723753448
Cl        1.05511100512666    0.06326947991444   -0.01596789615949
C        -0.72528016406076   -0.04222168979082    0.01338001029948
H        -0.72943166810099   -0.64186697484450   -0.90593862643410
H        -0.70325077485462   -0.53949486945042    0.99170851675593
H        -0.81467274532735    1.05024458819832   -0.04489924199632
 
 Bond Distances (Angstroems)
 ---------------------------
F1-C3=1.5030         Cl2-C3=1.7838        C3-F1=1.5030         C3-Cl2=1.7838        C3-H4=1.0976         C3-H5=1.0977         
C3-H6=1.0977         H4-C3=1.0976         H5-C3=1.0977         H6-C3=1.0977         
 C  H  Rav=1.0976 sigma=0.0000  Rmin=1.0976  Rmax=1.0977     3
 F  C  Rav=1.5030 sigma=0.0000  Rmin=1.5030  Rmax=1.5030     1
 Cl C  Rav=1.7838 sigma=0.0000  Rmin=1.7838  Rmax=1.7838     1
 
 selected bond angles (degree)
 --------------------
Cl2-C3-F1=179.90               H4-C3-F1= 88.82                H4-C3-Cl2= 91.28               H5-C3-F1= 88.71                
H5-C3-Cl2= 91.23               H5-C3-H4=119.94                H6-C3-F1= 88.73                H6-C3-Cl2= 91.23               
H6-C3-H4=119.93                H6-C3-H5=119.99                
 
 selected dihedral angles (degree)
 ---------------------------------

 
 CMA Distance (Angstroems)
 ---------------------------
R(CMA):  2.7541
           -------------------------------------------------
          |                Final Singlepoint                |
           -------------------------------------------------

          ...................................................
          :                      SETUP                      :
          :.................................................:
          :  # basis functions                  21          :
          :  # atomic orbitals                  20          :
          :  # shells                           10          :
          :  # electrons                        22          :
          :  max. iterations                   250          :
          :  Hamiltonian                  GFN2-xTB          :
          :  restarted?                      false          :
          :  GBSA solvation                  false          :
          :  PC potential                    false          :
          :  electronic temp.          300.0000000     K    :
          :  accuracy                    1.0000000          :
          :  -> integral cutoff          0.2500000E+02      :
          :  -> integral neglect         0.1000000E-07      :
          :  -> SCF convergence          0.1000000E-05 Eh   :
          :  -> wf. convergence          0.1000000E-03 e    :
          :  Broyden damping             0.4000000          :
          ...................................................

 iter      E             dE          RMSdq      gap      omega  full diag
   1    -13.1373331 -0.131373E+02  0.198E-04    7.86       0.0  T
   2    -13.1373331 -0.225189E-10  0.141E-04    7.86     290.2  T
   3    -13.1373331 -0.242260E-10  0.707E-05    7.86     577.4  T

   *** convergence criteria satisfied after 3 iterations ***

         #    Occupation            Energy/Eh            Energy/eV
      -------------------------------------------------------------
         1        2.0000           -0.8585678             -23.3628
       ...           ...                  ...                  ...
         5        2.0000           -0.3397294              -9.2445
         6        2.0000           -0.3131512              -8.5213
         7        2.0000           -0.2389902              -6.5033
         8        2.0000           -0.2389803              -6.5030
         9        2.0000           -0.2099361              -5.7127
        10        2.0000           -0.2099103              -5.7120
        11        2.0000           -0.1707918              -4.6475 (HOMO)
        12                          0.1180737               3.2129 (LUMO)
        13                          0.2486001               6.7648
        14                          0.2486007               6.7648
        15                          0.2606550               7.0928
        16                          0.2606753               7.0933
       ...                                ...                  ...
        20                          0.4364685              11.8769
      -------------------------------------------------------------
                  HL-Gap            0.2888654 Eh            7.8604 eV
             Fermi-level           -0.0263591 Eh           -0.7173 eV

 SCC (total)                   0 d,  0 h,  0 min,  0.002 sec
 SCC setup                      ...        0 min,  0.000 sec (  3.306%)
 Dispersion                     ...        0 min,  0.000 sec (  0.955%)
 integral evaluation            ...        0 min,  0.001 sec ( 32.858%)
 zeroth order Hamiltonian       ...        0 min,  0.000 sec (  1.161%)
 iterations                     ...        0 min,  0.001 sec ( 24.769%)
 molecular gradient             ...        0 min,  0.001 sec ( 33.379%)
 printout                       ...        0 min,  0.000 sec (  2.106%)

         :::::::::::::::::::::::::::::::::::::::::::::::::::::
         ::                     SUMMARY                     ::
         :::::::::::::::::::::::::::::::::::::::::::::::::::::
         :: total energy             -13.082694611964 Eh    ::
         :: gradient norm              0.000843890500 Eh/a0 ::
         :: HOMO-LUMO gap              7.860428981612 eV    ::
         ::.................................................::
         :: SCC energy               -13.137333132556 Eh    ::
         :: -> isotropic ES            0.164404275653 Eh    ::
         :: -> anisotropic ES         -0.019591396499 Eh    ::
         :: -> anisotropic XC         -0.037500327442 Eh    ::
         :: -> dispersion             -0.002483177254 Eh    ::
         :: repulsion energy           0.053823059699 Eh    ::
         :: add. restraining           0.000815430387 Eh    ::
         :::::::::::::::::::::::::::::::::::::::::::::::::::::

           -------------------------------------------------
          |                Property Printout                |
           -------------------------------------------------

    * Orbital Energies and Occupations

         #    Occupation            Energy/Eh            Energy/eV
      -------------------------------------------------------------
         1        2.0000           -0.8585678             -23.3628
         2        2.0000           -0.5512641             -15.0007
         3        2.0000           -0.4105356             -11.1712
         4        2.0000           -0.3397548              -9.2452
         5        2.0000           -0.3397294              -9.2445
         6        2.0000           -0.3131512              -8.5213
         7        2.0000           -0.2389902              -6.5033
         8        2.0000           -0.2389803              -6.5030
         9        2.0000           -0.2099361              -5.7127
        10        2.0000           -0.2099103              -5.7120
        11        2.0000           -0.1707918              -4.6475 (HOMO)
        12                          0.1180737               3.2129 (LUMO)
        13                          0.2486001               6.7648
        14                          0.2486007               6.7648
        15                          0.2606550               7.0928
        16                          0.2606753               7.0933
        17                          0.2929306               7.9710
        18                          0.4030713              10.9681
        19                          0.4032995              10.9743
        20                          0.4364685              11.8769
      -------------------------------------------------------------
                  HL-Gap            0.2888654 Eh            7.8604 eV
             Fermi-level           -0.0263591 Eh           -0.7173 eV

     #   Z        covCN         q      C6AA      α(0)
     1   9 F      0.768    -0.555    15.636     4.671
     2  17 Cl     0.920    -0.624   120.348    16.783
     3   6 C      4.426     0.234    16.907     5.839
     4   1 H      0.934    -0.018     3.397     2.882
     5   1 H      0.934    -0.018     3.397     2.882
     6   1 H      0.934    -0.018     3.397     2.882

 Mol. C6AA /au·bohr⁶  :        597.469955
 Mol. C8AA /au·bohr⁸  :      16392.713627
 Mol. α(0) /au        :         35.938524


Wiberg/Mayer (AO) data.
largest (>0.10) Wiberg bond orders for each atom
          total WBO             WBO to atom ...
     1  F   0.699        C    3 0.550    Cl   2 0.108
     2  Cl  0.615        C    3 0.465    F    1 0.108
     3  C   3.928        H    4 0.971    H    6 0.971    H    5 0.971    F    1 0.550    Cl   2 0.465
     4  H   1.000        C    3 0.971
     5  H   1.000        C    3 0.971
     6  H   1.000        C    3 0.971

molecular dipole:
                 x           y           z       tot (Debye)
 q only:        0.847       0.048      -0.019
   full:       -0.075      -0.007      -0.004       0.191
molecular quadrupole (traceless):
                xx          xy          yy          xz          yz          zz
 q only:      -11.819      -1.043       5.882       0.311       0.018       5.938
  q+dip:      -19.521      -1.728       9.714       0.504       0.030       9.807
   full:      -17.466      -1.546       8.691       0.452       0.027       8.774

           -------------------------------------------------
          |                Geometry Summary                 |
           -------------------------------------------------

      molecular mass/u    :       69.4858989
   center of mass at/Å    :       -0.2280909      -0.0124655       0.0058091
  moments of inertia/u·Å² :        0.3641501E+01   0.1402980E+03   0.1402995E+03
rotational constants/cm⁻¹ :        0.4629309E+01   0.1201559E+00   0.1201546E+00

 * 4 selected distances

     #   Z        #   Z                                     value/Å
     2  17 Cl     3   6 C                                 1.7837551 (max)
     3   6 C      4   1 H                                 1.0976058 (min)
     3   6 C      5   1 H                                 1.0976760
     3   6 C      6   1 H                                 1.0976657

 * 2 distinct bonds (by element types)

   Z      Z             #   av. dist./Å        max./Å        min./Å
   1 H    6 C           3     1.0976492     1.0976760     1.0976058
   6 C   17 Cl          1     1.7837551     1.7837551     1.7837551


optimized geometry written to: xtbopt.xyz


           -------------------------------------------------
          | TOTAL ENERGY              -13.082694611964 Eh   |
          | GRADIENT NORM               0.000843890500 Eh/α |
          | HOMO-LUMO GAP               7.860428981612 eV   |
           -------------------------------------------------

------------------------------------------------------------------------
 * finished run on 2020/04/08 at 23:04:04.735     
------------------------------------------------------------------------
 total:
 * wall-time:     0 d,  0 h,  0 min,  0.033 sec
 *  cpu-time:     0 d,  0 h,  0 min,  0.033 sec
 * ratio c/w:     0.999 speedup
 SCF:
 * wall-time:     0 d,  0 h,  0 min,  0.003 sec
 *  cpu-time:     0 d,  0 h,  0 min,  0.003 sec
 * ratio c/w:     0.999 speedup
 ANC optimizer:
 * wall-time:     0 d,  0 h,  0 min,  0.023 sec
 *  cpu-time:     0 d,  0 h,  0 min,  0.023 sec
 * ratio c/w:     1.000 speedup

